src/Smidge.Core/CompositeFiles/DefaultUrlManager.cs

```diff
@@ -160,8 +162,8 @@ public ParsedUrlPath ParsePath(string input)
         private string GetCompositeUrl(string fileKey, string fileExtension, string cacheBusterValue)
         {
             //Create a delimited URL query string
-
-            const string handler = "~/{0}/{1}{2}.v{3}";
+			
```

Comment:
Undo.

Reply:
Removed the new line on 165. Had to do this outside of Visual Studio.